### PR TITLE
Added :hover to nav__link in header.scss, :hover to catalog__card in catalog.scss

### PR DESCRIPTION
**Pull request by yurakosh701**
- [DEMO LINK](https://yurakosh701.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
```
Comment (polosanya) [inline src/styles/blocks/catalog.scss:20]: But you also change color on hover, don't you need to add it here?

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```
Comment (polosanya) [inline src/styles/blocks/header.scss:41]: Please specify to which properties you want to apply the transition. 
For now, it applies to all the properties, and it's really bad for performance

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done! Please check the comments

**Comment by Bohdan-Maksymiuk** [comment]
You don't need to close PR's

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
apply changes and re-request review

**Comment by yurakosh701** [comment]
I closed PR, because I created new 
https://github.com/mate-academy/layout_catalog_hovers/pull/1300

**Comment by yurakosh701** [comment]
> apply changes and re-request review

I closed PR, because I created new 
https://github.com/mate-academy/layout_catalog_hovers/pull/1300

**Pull request closed without merging**